std/traits.d

```diff
@@ -7097,19 +7097,49 @@ if (T.length == 1)
 }
 
 /**
-Determines whether function $(D f) requires a context pointer.
+Determines whether function `f` requires a context pointer.
+
+Params:
+    f = a function
```

Comment:
If evaluating `isNestedFunction` for a non-function parameter is okay then the parameter docs should say so. If it's not okay then it should be a template constraint rather than part of the body.